contrib/renew-root-cert

```diff
@@ -0,0 +1,49 @@
+#!bin/sh -e
+
+# This is an OnChanged handler that can handle requests to renew
+# the factory root CA certificate used to verify the device gateway TLS certificate.
+
+[ -z "$CONFIG_FILE" ] && (echo "No CONFIG_FILE specified"; exit 1)
+[ -f "$CONFIG_FILE" ] || (echo "$CONFIG_FILE does not exist"; exit 1)
+[ -z "$SOTA_DIR" ] && (echo "No SOTA_DIR specified"; exit 1)
```

Comment:
This variable does not seem to be used anymore, or is  it supposed to be `STORAGE_DIR` that is references later on ? :+1:

Reply:
Then do we need to check if it is actually a directory with `[ -d $SOTA_DIR ] || ` ?

Reply:
This is a copy paste from the other script: https://github.com/foundriesio/fioconfig/blob/main/contrib/renew-client-cert#L8

I think you are right and this code needs to check for the STORAGE_DIR instead.
It needs to be fixed in both the original and the copy.

It is a minor fix, and things _just work_ in old versions (with this mistake); so no need to backport.

Reply:
Good catch! 🚀